src/tasksInfo/index.js

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react'
+
+import { getInfo } from '../contracts/getInfo'
+import { checkUser } from '../contracts/checkUser'
+import { getTasksInfo } from '../helpers/constansInfo'
+import styles from './styles.css'
+
+
+export const TasksInfo = () => {
+  const [info, setInfo] = useState({})
+  const [user, setUser] = useState('')
+
+  useEffect(() => {
+    const getData = async () => {
+      setInfo(await getInfo())
+    }
+
+    getData()
+
+    const getName = async () => {
+      setUser(await checkUser())
+    }
+
+    getName()
```

Comment:
why you don't do it in a one function?

Reply:
I thought it would look better this way. Now I will change it into one function